src/Provider/SessionProvider.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Provider;
+
+use SessionHandlerInterface;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\Support\ServiceProvider;
+use Yiisoft\Yii\Web\Session\Session;
+use Yiisoft\Yii\Web\Session\SessionInterface;
+
+final class SessionProvider extends ServiceProvider
+{
+    private static array $sessionOptions;
+    private static ?SessionHandlerInterface $sessionHandler;
+
+    public function __construct(
+        array $sessionOptions = [['cookie_secure' => 0]],
```

Comment:
It is likely to be changed in the application config.

Reply:
yes, options its default, in case you don't want to define it.